vm-bootstorm/azuredeploy.json

```diff
@@ -0,0 +1,366 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.15.11.2",
+  "parameters": {
+    "azureAdApplicationId": {
+      "type": "string",
+      "metadata": {
+        "description": "Azure AD Application ID to authenticate inside controller VM."
+      }
+    },
+    "azureAdApplicationPassword": {
+      "type": "string",
+      "metadata": {
+        "description": "Azure AD Application password to authenticate inside controller VM."
+      }
+    },
+    "tenantId": {
+      "type": "string",
```

Comment:
Hmm, I though this function returned but it doesn't look like it https://azure.microsoft.com/en-us/documentation/articles/resource-group-template-functions/#subscription